core/server/worker/src/test/java/alluxio/worker/block/BlockStoreMetaTest.java

```diff
@@ -59,12 +59,12 @@ public void before() throws Exception {
       TieredBlockStoreTestUtils.cache(TEST_SESSION_ID, blockId, TEST_BLOCK_SIZE, dir,
           mMetadataManager, null);
     }
-    mBlockStoreMeta = BlockStoreMeta.getBlockStoreMeta(mMetadataManager);
-    mBlockStoreMetaFull = BlockStoreMeta.getBlockStoreMetaFull(mMetadataManager);
+    mBlockStoreMeta = BlockStoreMeta.Factory.getBlockStoreMeta(mMetadataManager);
+    mBlockStoreMetaFull = BlockStoreMeta.Factory.getBlockStoreMetaFull(mMetadataManager);
   }
 
   /**
-   * Tests the {@link BlockStoreMeta#getBlockList()} method.
+   * Tests the {@link DefaultBlockStoreMeta#getBlockList()} method.
```

Comment:
revert javadoc change, as we are testing methods of `BlockStoreMeta`, not `DefaultBlockStoreMeta`